.github/workflows/cpplint-scan.yml

```diff
@@ -0,0 +1,59 @@
+name: "Cpplint Scan"
+
+on:
+  push:
+    branches: [master, devel]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master, devel]
+
+jobs:
+  cppcheck:
```

Comment:
This identifier doesn't seem to show up in the logs/UI anywhere but let's also rename it cpplint to avoid confusion

Reply:
Agreed and done!